pkg/kubelet/volumemanager/reconciler/reconciler_test.go

```diff
@@ -1097,16 +1097,14 @@ func Test_Run_Positive_VolumeFSResizeControllerAttachEnabled(t *testing.T) {
 				},
 			}
 
-			// deep copy before reconciler runs to avoid data race.
-			pvWithSize := pv.DeepCopy()
```

Comment:
Why did you remove it? Let's not try mix changes like this in huge PRs like this.